IpmiFeaturePkg/Library/IpmiBaseLibMm/IpmiBaseLibMm.inf

```diff
@@ -0,0 +1,33 @@
+## @file
+#
+# @copyright
+# Copyright 2010 - 2021 Intel Corporation. <BR>
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION                    = 1.26
+  PI_SPECIFICATION_VERSION       = 0x00010032
+  BASE_NAME                      = IpmiBaseLibMm
+  FILE_GUID                      = 06C28B6D-F15F-42C3-8316-01BD3E93D2DE
+  MODULE_TYPE                    = MM_STANDALONE
+  VERSION_STRING                 = 1.0
+  LIBRARY_CLASS                  = IpmiBaseLib|MM_STANDALONE MM_CORE_STANDALONE

```

Comment:
Why is this a base lib but still only used for mm?

Reply:
It is the original naming convention as below picture, I think it does not mean `BASE` type here, it should mean it is a basic lib that provide basic API for SEL logging, but it needs to locate IpmiTransport PPI/Protocol/SMM Protocol/MM Protocol according to the driver type accordingly.
![image](https://github.com/user-attachments/assets/7c70042a-c9e4-407b-8cbf-0589072709e0)